pucauto.py

```diff
@@ -14,8 +14,20 @@
 with open("config.json") as config:
     CONFIG = json.load(config)
 
-
-DRIVER = webdriver.Firefox()
+if CONFIG['proxy'][0]['server'] and CONFIG['proxy'][0]['port']:
```

Comment:
If you make `proxy` and object in the config, all of the `['proxy'][0]` references can just be `['proxy']['key']`.